### PR TITLE
Fix: file transfer auditlogs

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Review by merlin-northern — APPROVED** [review]

File: api/http/management_filetransfer.go [hunk 1]
```diff
@@ -407,19 +407,22 @@ func (h ManagementController) downloadFileResponseProcessMessage(c *gin.Context,
 func (h ManagementController) DownloadFile(c *gin.Context) {
 	params, statusCode, err := h.getFileTransferParams(c)
 	if err != nil {
+		log.FromContext(c).Error(err)
```
Comment (alfrunes) [inline api/http/management_filetransfer.go:410]: FYI: `gin.Context != gin.Context.Request.Context`. IOW you won't get the values attached to the request context (request id, subject id etc.).
And can you please create a logger variable at the start of the function?

Comment (tranchitella) [reply api/http/management_filetransfer.go]: Thanks @alfrunes, I force-pushed, can you check again?

**Review by alfrunes — APPROVED** [review]

**Comment by tranchitella** [comment]
Not running the integration tests again, as the only change since the previous commit are the log statements.

**Pull request merged**